src/app/shared/models/student.model.ts

```diff
@@ -0,0 +1,10 @@
+import { Item } from './item.model';
+
+export class Student {
+  id: number;
```

Comment:
I like that we're changing the model members. Before you merge, please update the item model to match this for consistency.

Reply:
will do after work